### PR TITLE
Clarifying the use of both `Project Groups`and `Projects`  filters

### PR DESCRIPTION
**Pull request by johnsimons**
When combining both of these filters things can be difficult to understand.
These PR tries to address this issue by using examples

### CODE REVIEW

File: docs/administration/managing-users-and-teams/index.md [hunk 1]
```diff
@@ -40,14 +40,33 @@ Invite codes are only valid for 48-hours aftering being generated, so make sure
...
-Creating teams lets you assign the same roles to groups of users. Users can be added to or removed from multiple teams, making it eaiser to manage permissions for specific users and teams. You can create new teams by using the **Add Team** button.
+Creating teams lets you assign the same roles to groups of users. Users can be added to or removed from multiple teams, making it easier to manage permissions for specific users and teams. You can create new teams by using the **Add Team** button.
 
 For example, we can create a team that gives Anne and Bob access to view projects and deploy them to pre-production environments by assigning the role **Project deployer** to the team.
 
 We limit which projects and environments these permissions apply to by adding specific projects and environments to the team.
 
 ![](teams-create-new.png "width=500")
 
+### Locking down which projects you can see
```
Comment (johnsimons) [inline docs/administration/managing-users-and-teams/index.md]: Not entirely happy with the heading name, @NickJosevski @leecarlon suggestions ?

Comment (KennethBates) [reply docs/administration/managing-users-and-teams/index.md]: Maybe:
### Restricting project and project group access
?

File: docs/administration/managing-users-and-teams/index.md [hunk 2]
```diff
@@ -40,14 +40,33 @@ Invite codes are only valid for 48-hours aftering being generated, so make sure
...
+
+The following table illustrates the combination of possible permissions when specifying both `Project Groups` and `Projects` filters:
+
+| `Project Groups` | `Projects` | Result                                   |
+| ---------------- | ---------- | ---------------------------------------- |
+| `Empty`          | `Empty`    | Project1, Project2, Project3, Project4 and Project5 |
+| `Empty`          | Project1   | Project1                                 |
+| GroupB           | `Empty`    | Project4 and Project5                    |
+| GroupA           | Project5   | Project1, Project2, Project3 and Project5 |
+| GroupB           | Project4   | Project4 and, Project5                   |
```
Comment (KennethBates) [inline docs/administration/managing-users-and-teams/index.md]: `Project4 and Project5`
Removing the comma after the 'and'

File: docs/administration/managing-users-and-teams/index.md [hunk 3]
```diff
@@ -40,14 +40,33 @@ Invite codes are only valid for 48-hours aftering being generated, so make sure
...
 
 For example, we can create a team that gives Anne and Bob access to view projects and deploy them to pre-production environments by assigning the role **Project deployer** to the team.
 
 We limit which projects and environments these permissions apply to by adding specific projects and environments to the team.
 
 ![](teams-create-new.png "width=500")
 
+### Locking down which projects you can see
+
+When specifying both `Project Groups`and `Projects`  filters please be aware that both these filters complement each other, to better illustrate these filters in action lets consider the following project structure:
```
Comment (KennethBates) [inline docs/administration/managing-users-and-teams/index.md]: When specifying both `Project Groups` and `Projects` filters please be aware that both these filters complement each other. To better illustrate these filters in action let's consider the following project structure:

File: docs/administration/managing-users-and-teams/index.md (hunk 1 above)
Comment (leecarlon) [reply docs/administration/managing-users-and-teams/index.md]: +1 to Restricting project and project group access

**Pull request merged**